mozilla_schema_generator/generic_ping.py

```diff
@@ -24,10 +24,13 @@ class GenericPing(object):
     extra_schema_key = "extra"
     cache_dir = pathlib.Path(".probe_cache")
 
-    def __init__(self, schema_url, env_url, probes_url):
+    def __init__(self, schema_url, env_url, probes_url, mps_branch=None):
         self.schema_url = schema_url
         self.env_url = env_url
         self.probes_url = probes_url
+        if mps_branch is not None:
```

Comment:
This seems brittle (for when e.g. the url does not contain master), and also pushes the default branch now to the url rather than to the code here. Instead we should add a placeholder in the url, and always replace it with the branch, which can have a default value of `master`.